patches/minecraft/net/minecraft/client/renderer/BlockRendererDispatcher.java.patch

```diff
@@ -41,16 +41,23 @@
        BlockRenderType blockrendertype = p_228791_1_.func_185901_i();
        if (blockrendertype != BlockRenderType.INVISIBLE) {
           switch(blockrendertype) {
-@@ -89,7 +101,7 @@
+@@ -89,10 +101,13 @@
              float f = (float)(i >> 16 & 255) / 255.0F;
              float f1 = (float)(i >> 8 & 255) / 255.0F;
              float f2 = (float)(i & 255) / 255.0F;
 -            this.field_175027_c.func_228804_a_(p_228791_2_.func_227866_c_(), p_228791_3_.getBuffer(RenderTypeLookup.func_228394_b_(p_228791_1_)), p_228791_1_, ibakedmodel, f, f1, f2, p_228791_4_, p_228791_5_);
 +            this.field_175027_c.renderModel(p_228791_2_.func_227866_c_(), p_228791_3_.getBuffer(RenderTypeLookup.func_228394_b_(p_228791_1_)), p_228791_1_, ibakedmodel, f, f1, f2, p_228791_4_, p_228791_5_, modelData);
              break;
           case ENTITYBLOCK_ANIMATED:
-             ItemStackTileEntityRenderer.field_147719_a.func_228364_a_(new ItemStack(p_228791_1_.func_177230_c()), p_228791_2_, p_228791_3_, p_228791_4_, p_228791_5_);
-@@ -101,4 +113,9 @@
+-            ItemStackTileEntityRenderer.field_147719_a.func_228364_a_(new ItemStack(p_228791_1_.func_177230_c()), p_228791_2_, p_228791_3_, p_228791_4_, p_228791_5_);
++            {
++               ItemStack stack = new ItemStack(p_228791_1_.func_177230_c());
++               stack.func_77973_b().getTileEntityItemStackRenderer().func_228364_a_(stack, p_228791_2_, p_228791_3_, p_228791_4_, p_228791_5_);
```

Comment:
This needs to be updated now that #6461 got merged, as it is no longer `getTileEntityItemStackRenderer` but is instead `getItemStackTileEntityRenderer`